code/Frontend/react-frontend/src/pages/AddItem.js

```diff
@@ -12,36 +12,61 @@ function EquipmentForm() {
     price: "",
     user: "",
   });
+  function getUserId() {
+    fetch("/api/userInfo", {
+      method: "GET",
+      credentials: "include", // Include cookies in the request if using cookies for sessions
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        const userId = data.userId; // Extract the user ID from the response data
+        console.log("User ID:", userId);
+      })
+      .catch((error) => console.error("Error:", error));
+  }
 
-  const handleInputChange = (event) => {
-    const { name, value } = event.target;
-    setFormData({ ...formData, [name]: value });
-  };
+  const handleSubmit = () => {
+    // Get the user ID obtained during login
+    const userId = getUserId(); // Implement a function to retrieve the user ID
+
+    // Create the equipment object with the user's ID as the owner
+    const newEquipment = {
+      name: formData.name,
+      description: formData.description,
+      status: formData.status,
+      price: formData.price,
+      owner: userId, // Include the user's ID as the owner
+    };
 
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    // Send the formData as JSON to your Flask back-end here
-    fetch("http://127.0.0.1:5000/api/addEquipment", {
+    // Send a POST request to your API to create the equipment
+    fetch("http://your-api-url/api/addEquipment", {
```

Comment:
How does this work? I assume your-api-url is a variable created somewhere but I can't find it.

Reply:
Yeah this was a mistake - it shouldn’t work - I will fix it